cpmpy/solvers/pysat.py

```diff
@@ -243,100 +190,124 @@ def _after_solve(self, pysat_status):
-                # only handle cardinality encodings
-                if isinstance(con.args[0], Operator) and con.args[0].name == "sum" and all(isinstance(v, _BoolVarImpl) for v in con.args[0].args):
-                    lits = [self.pysat_var(var) for var in con.args[0].args]
-                    bound = con.args[1]
-                    #TODO: Edge case where sum(x) < 0: Raises error
-                    if con.name == "<":
-                        atmost = CardEnc.atmost(lits=lits, bound=bound - 1, vpool=self.pysat_vpool)
-                        cnf.extend(atmost.clauses)
-                    elif con.name == "<=":
-                        atmost = CardEnc.atmost(lits=lits, bound=bound, vpool=self.pysat_vpool)
-                        cnf.extend(atmost.clauses)
-                    elif con.name == ">=":
-                        atleast = CardEnc.atleast(lits=lits, bound=bound, vpool=self.pysat_vpool)
-                        cnf.extend(atleast.clauses)
-                    elif con.name == ">":
-                        atleast = CardEnc.atleast(lits=lits, bound=bound+1, vpool=self.pysat_vpool)
-                        cnf.extend(atleast.clauses)
-                    elif con.name == "==":
-                        equals = CardEnc.equals(lits=lits, bound=bound, vpool=self.pysat_vpool)
-                        cnf.extend(equals.clauses)
-                    # special cases with bounding 'hardcoded' for clarity
-                    elif con.name == "!=" and bound <= 0:
-                        atleast = CardEnc.atleast(lits=lits, bound=bound+1, vpool=self.pysat_vpool)
-                        cnf.extend(atleast.clauses)
-                    elif con.name == "!=" and bound >= len(lits):
-                        atmost = CardEnc.atmost(lits=lits, bound=bound - 1, vpool=self.pysat_vpool)
-                        cnf.extend(atmost.clauses)
-                    elif con.name == "!=":
-                        ## add implication literal
-                        is_atleast = self.pysat_var(boolvar())
-                        atleast = [cl + [-is_atleast] for cl in CardEnc.atleast(lits=lits, bound=bound+1, vpool=self.pysat_vpool).clauses]
-                        cnf.extend(atleast)
-                        
-                        is_atmost = self.pysat_var(boolvar())
-                        atmost =  [cl + [-is_atmost] for cl in CardEnc.atmost(lits=lits, bound=bound-1, vpool=self.pysat_vpool).clauses]
-                        cnf.extend(atmost)
-                        
-                        ## add is_atleast or is_atmost
-                        cnf.append([is_atleast, is_atmost])
-                    else:
-                        raise NotImplementedError(f"Non-operator constraint {con} not supported by CPM_pysat")
-                else:
-                    raise NotImplementedError(f"Non-operator constraint {con} not supported by CPM_pysat")
+            CPMpy will return only those variables that are False (in the UNSAT core)
```

Comment:
not related to the unification, but 'those assumption variables that are False' would be more explicit...

Also, the next line, the 'though this interf...' part is now in CPMpy so can be cut.

Reply:
Ok, added comment to refer to examples folder
In the future: refer to "tools-like" folder?